README.md

```diff
@@ -1 +1,18 @@
-# .github
\ No newline at end of file
+# BiomedSci AI organization
+## 🙋‍♀️ What is BiomedSciAI?
+
+Our mission is to accelerate scientific discovery in Biomedical Sciences through a variety of tools that can help scientists do their research quicker and more effectively.
+
+Whether you use population-scale data like EHRs, subcellullar-scale data like spatially resolved omics, or a combination of various modalities, we host tools that will help you make the most out of the data. 
+
+## 🌈 How can I get involved?
+
+If you want to contribute and get involved check out our guidelines and code of conduct.
```

Comment:
We don't have any yet. Do we plan to add these as part of this PR?

Reply:
I was planning to merge only once we have the rest of the things in place

Reply:
FYI if you intend to add more things to the PR, it is best to [mark it as draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft), to signalize it is not finalized.